CMakeLists.txt

```
@@ -1,5 +1,5 @@
-cmake_minimum_required(VERSION 2.8.1)
-project(UnitTest++)
+cmake_minimum_required(VERSION 3.0)
+project(UnitTest++ VERSION 2.1)
```

Let's make the version explicitly 3 parts. I agree that changing the CMake requirement merits a minor version bump, though, not a patch bump, so `2.1.0`:

```suggestion
project(UnitTest++ VERSION 2.1.0)
```

I'm not aware of CMake supporting `-pre` or `-beta` suffixes to distinguish this from a release version but I haven't been active with CMake in years.

---

Thanks.  Just made and pushed the changes requested.